05.wc/wc.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require 'optparse'
+
+def main
+  options = define_options
+
+  if stand_alone?
+    files = ARGV
+    file_details = files.to_h { |file| [file, File.read(File.expand_path(file))] }
+    file_details_total = Hash.new(0)
+
+    file_details.each do |file_name, file_text|
+      detail_line = create_details(file_text, options)
+      puts "#{add_space(detail_line)} #{file_name}"
+
+      options.each do |option|
+        file_details_total[option] += detail_line[option]
+      end
+    end
+
+    puts "#{add_space(file_details_total)} total" if file_details.size > 1
+  else
+    detail_line = create_details($stdin.to_a.join, options)
+    puts add_space(detail_line)
+  end
+end
+
+def create_details(file_text, options)
```

Comment:
```suggestion
def create_details(text, options)
```

文字列ならなんでもいいはずなので file_ はなくしちゃいましょう。

> ただし、必ず一致させないといけないわけではないので、機械的に対応しないようにしてください。（メソッド定義側の引数名は多少抽象度が上がっても大丈夫なことが多い）
> https://bootcamp.fjord.jp/pages/readable-variable-name